pyart/retrieve/tests/test_kdp_proc.py

```diff
@@ -7,7 +7,24 @@
 from pyart.testing import sample_objects
 from pyart.config import get_field_name
 
+def test_kdp_vulpiani_linear_psidp(slope=0.002):
+    radar = _make_linear_psidp_radar(slope=slope)
+    kdp_dict, phidp_dict = kdp_proc.kdp_vulpiani(radar,windsize=10)
+    valid_kdp = kdp_dict['data'][0][np.isfinite(kdp_dict['data'][0])]
+    assert np.allclose(np.diff(valid_kdp), 0.0, atol=0.1)
```

Comment:
The first five values in the first ray of kdp_dict['data'] are -- (masked) 0.001, 0.001, 0.001 and 0.001 which seem incorrect.  Please investigate.